content/applications/sales/crm/performance/win_loss.rst

```diff
@@ -1,26 +1,249 @@
+include:
+
+- **Created On**: Adjusting this filter to a different period of time such as the last 30 days
+  or the last quarter can provide more timely results.
+- **Add Custom Filter**: Clicking :guilabel:`Add Custom Filter` and scrolling through the numerous
+  options in the drop-down opens up additional search criteria like :guilabel:`Last Stage Update` or
+  :guilabel:`Lost Reason`.
+- **Multiple Groupings**: Adding multiple *Group By* selections can split results into more
+  relevant and manageable chunks.
+
+  - Adding :guilabel:`Salesperson` or :guilabel:`Sales Team` breaks up the total count of leads in
+    each :guilabel:`Stage` by who they were assigned to.
+  - Adding :guilabel:`Medium` or :guilabel:`Source` can reveal what marketing avenues generate more
+    sales.
+- **Add Custom Group > Active**: Clicking :menuselection:`Add Custom Group --> Active` separates the
+  results in each stage into *Won (true)* or *Lost (false)*. This can show at what stage leads are
+  won and lost.
+
+.. image:: win_loss/search-panel-filters-and-group-by-options.png
+   :align: center
+   :alt: The Search menu open and the Won and Lost filters highlighted.
+
+Additional measurements and views
+---------------------------------
+
+In addition to filters and group by options, the *Pipeline Analysis* can also display different
+measurements, chart types, and :ref:`report views <reporting/views>`. Need to know something other
+than the total count of leads? Change the measurement to something more valuable. Want to see the
+report as a spreadsheet? Switch to the :guilabel:`Pivot View`.
+
+To change the measurement, click the :guilabel:`Measures` button and select an option from the
+drop-down such as :guilabel:`Expected Revenue`, :guilabel:`Days to Assign`, or
+:guilabel:`Days to Close`.
+
+.. image:: win_loss/measure-button-menu.png
+   :align: center
+   :alt: The Measures drop-down menu showing Count, Expected Revenue, and other options.
+
+In addition to the standard :guilabel:`Bar Chart`, the graph icons along the top of the report can
+switch the visual to a :guilabel:`Line Chart` or :guilabel:`Pie Chart`. The :guilabel:`Stacked` icon
+further separates the results in a :guilabel:`Bar Chart` or :guilabel:`Line Chart`.
+
+.. image:: win_loss/graph-icons.png
+   :align: center
+   :alt: The graph icons along the top of the report change how the results are displayed visually.
+
+Finally, the :ref:`four icons in the top-right <reporting/views>` provide more detailed views of the
+numbers and records that make up the report. Instead of the default :guilabel:`Graph View`, try
+clicking the :guilabel:`Pivot View` to see the report in a table format, or :guilabel:`List View`
+to see a single list of every record in the report.
+
+.. image:: win_loss/pivot-and-list-views.png
+   :align: center
+   :alt: The same win/loss report displays different information when using Pivot View or List View.
+
+Win/Loss in pivot view
+======================
```

Comment:
This section may be better served above, if you take on the approach suggested -- wherein you break down every element of the Pipeline Analysis page (including all the different view options)

Reply:
And the following portion about inserting it into a spreadsheet would be a great addition, too